common-bom/pom.xml

```diff
@@ -13,16 +13,16 @@
     <name>Common BOM</name>
     <description>Common bill of materials defining standard versions for dependencies</description>
 
-    <!-- ===== NOTES ===== The dependencies defined here are NOT included in
```

Comment:
not sure if this document has been formatted at all. there were a few elements with quite random indentation